plugins/periskop/src/components/EntityPeriskopErrorsCard/EntityPeriskopErrorsCard.tsx

```diff
@@ -0,0 +1,187 @@
+    return <StatusError>{severity}</StatusError>;
+  }
+  return <StatusPending>{severity}</StatusPending>;
+};
+
+const renderLastOccurrence = (error: AggregatedError): React.ReactNode => {
+  return DateTime.fromMillis(
+    error.latest_errors[0].timestamp * 1000,
+  ).toRelative();
+};
+
+function isNotFoundInInstance(
+  apiResult: AggregatedError[] | NotFoundInInstance | undefined,
+): apiResult is NotFoundInInstance {
+  return (apiResult as NotFoundInInstance)?.body !== undefined;
+}
+
+export const EntityPeriskopErrorsCard = () => {
+  const { entity } = useEntity();
+  const entityPeriskopName: string =
+    entity.metadata.annotations?.[PERISKOP_NAME_ANNOTATION] ??
+    entity.metadata.name;
+
+  const periskopApi = useApi(periskopApiRef);
+  const instanceNames = periskopApi.getInstanceNames();
+  const [instanceOption, setInstanceOption] = React.useState<string>(
+    instanceNames[0],
+  );
+  const {
+    value: aggregatedErrors,
+    loading,
+    error,
+  } = useAsync(async (): Promise<AggregatedError[] | NotFoundInInstance> => {
+    return periskopApi.getErrors(instanceOption, entityPeriskopName);
+  }, [instanceOption]);
+
+  if (loading) {
+    return <Progress />;
+  } else if (error) {
+    return <ResponseErrorPanel error={error} />;
+  }
+
+  const columns: TableColumn<AggregatedError>[] = [
+    {
+      title: 'Key',
+      field: 'aggregation_key',
+      highlight: true,
+      sorting: false,
+      render: aggregatedError => {
+        const errorUrl = periskopApi.getErrorInstanceUrl(
+          instanceOption,
+          entityPeriskopName,
+          aggregatedError,
+        );
+        return renderKey(aggregatedError, errorUrl);
+      },
+    },
+    { title: 'Occurrences', field: 'total_count', sorting: true },
+    {
+      title: 'Last Occurrence',
+      render: aggregatedError => renderLastOccurrence(aggregatedError),
+      defaultSort: 'asc',
+      customSort: (a, b) =>
+        b.latest_errors[0].timestamp - a.latest_errors[0].timestamp,
+      type: 'datetime',
+    },
+    {
+      title: 'Severity',
+      field: 'severity',
+      render: aggregatedError => renderSeverity(aggregatedError.severity),
+      sorting: false,
+    },
+  ];
+
+  const sortingSelect = (
+    <Select
+      selected={instanceOption}
+      label="Instance"
+      items={instanceNames.map(e => ({
+        label: e,
+        value: e,
+      }))}
+      onChange={el => setInstanceOption(el.toString())}
+    />
+  );
+
+  const contentHeader = (
+    <ContentHeader title="Periskop" description={entityPeriskopName}>
+      {sortingSelect}
+    </ContentHeader>
+  );
+
+  if (isNotFoundInInstance(aggregatedErrors)) {
+    return (
+      <Content noPadding>
+        {contentHeader}
+        <EmptyState
+          title="Periskop returned an error"
+          description={aggregatedErrors.body}
+          missing="data"
+        />
+      </Content>
+    );
+  }
+
+  return (
+    <Content noPadding>
```

Comment:
was a bit surprised that this didn't render an actual `<Card>`, but just the contents - that's commonly what's done